gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+## Title
+
+* Gig Marketplace
+
+## Patlet
+
+InnerSource projects at your company are finding it difficult to attract contributors.  
+
+Establish a marketplace by creating an intranet website that list specific InnerSource project needs as "gigs" with explicit time and skill requirements.
+This will enable managers to better understand their employee’s time commitment and professional benefits thereby increasing the likelihood of garnering approval to make InnerSource contributions.
+
+## Problem
+
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other. 
+It is difficult for employees to communicate to their management what commitment of time they will need to make to an InnerSource project and how they will benefit.  
```

Comment:
> how they will benefit

How does the gig marketplace help to explain the benefit back to the contributor?  Not clear.

Reply:
True.  The way it is presented it is just taken as a given that taking on the additional work of a Gig will somehow personally benefit the developer.  The marketplace website should list out potential benefits explicitly in its interface or documentation.  Also, on top of the time and skill requirements those listing the Gigs should include language in the Gig description stating what could potentially be learnt or the kind of impact someone could have by taking on the Gig.